docker-compose.yml

```diff
@@ -1,20 +1,15 @@
-version: '2'
+version: '2.4'
```

Comment:
Is any specific change dependent on version 2.4? I ask for my own understanding, not as a critique.

Reply:
I suspected that. Thanks for confirming.